evap/evaluation/management/commands/reload_testdata.py

```diff
@@ -36,4 +36,7 @@ def handle(self, *args, **options):
         self.stdout.write('Executing "python manage.py refresh_results_cache"')
         call_command("refresh_results_cache")
 
+        self.stdout.write('Executing "python manage.py clear_cache --cache="sessions""')
+        call_command('clear_cache --cache="sessions"')
```

Comment:
you can leave out the quotes around `sessions` here and in the comment to make this less confusing.